### DISCUSSION TITLE
Isar defaultMaxSizeMiB use

### DISCUSSION THREAD

**Discussion by abhay-s-rawat** [isar/isar#1291] [Help] [answered]
defaultMaxSizeMiB will be the max DB size ?

**Comment by simc** [comment]
Yes

**Reply by abhay-s-rawat** [reply]
After that size what will happpen ?

**Reply by abhay-s-rawat** [reply]
will it reject put commands ?

**Reply by abhay-s-rawat** [reply]
can we set it to null or infinite ?

**Reply by simc** [reply, marked as answer]
Yes you will get a database full error after a put(). On Desktop you can set it to a very high number (like 1TB) but phones have a limited amount of virtual memory so you should set it to something around 1GB at most. Better a little less. Isar is extremely space efficient and you can store millions of objects before you hit the limit
